google/cloud/servicemanagement/README.md

```diff
@@ -3,7 +3,9 @@
 :construction:
 
 This directory contains an idiomatic C++ client library for the
-[Service Management API][cloud-service-docs], a service to Google Service Management allows service producers to publish their services on Google Cloud Platform so that they can be discovered and used by service consumers.
+[Service Management API][cloud-service-docs]. This service allows service
+producers to publish their services on Google Cloud Platform so that they can be
+discovered and used by service consumers.
```

Comment:
No change. Just highlighting a funny sentence that uses the word "service" 4 times.

Reply:
I was going to Say "Allows service producers" ... but somebody told me that was not a complete sentence 😁